### ISSUE TITLE
wrong css class

### ISSUE THREAD

**Issue by olegdovger** [ember-cli/ember-cli#4673]
I have got this 
```<div id="ember821" class="pull-right p u l l - r i g h t ef-daterange">```

after I set class for my component

```{{ef-daterange
      value=dateRangeValue
      classNames="pull-right"
      format="dd, DD.MM.YYYY"}}```

What is wrong?

**Comment by stefanpenner** [comment]
Hello, this appears to be the wrong repo. This is something likely to be reported on ember.js.

But i can save you time, `classNames` -> `class` should fix your issue.

**Comment by rwjblue** [comment]
It is also fixed with Ember 1.13.8.

**Comment by stefanpenner** [comment]
> It is also fixed with Ember 1.13.8.

:+1:
